kloppy/infra/serializers/event/wyscout/deserializer_v3.py

```diff
@@ -152,14 +152,20 @@ def _check_secondary_event_types(
 def _pass_qualifiers(raw_event) -> List[Qualifier]:
     qualifiers = _generic_qualifiers(raw_event)
 
-    if _check_secondary_event_types(raw_event, ["cross", "cross_blocked"]):
-        qualifiers.append(PassQualifier(PassType.CROSS))
-    elif _check_secondary_event_types(raw_event, ["hand_pass"]):
-        qualifiers.append(PassQualifier(PassType.HAND_PASS))
-    elif _check_secondary_event_types(raw_event, ["head_pass"]):
-        qualifiers.append(PassQualifier(PassType.HEAD_PASS))
-    elif _check_secondary_event_types(raw_event, ["smart_pass"]):
-        qualifiers.append(PassQualifier(PassType.SMART_PASS))
+    qualifier_mapping = {
```

Comment:
How about swapping the keys and values in the dictionary? That change would avoid the need to cast the tuples into lists before passing the qualifiers to the `_check_secondary_event_types` function.